packages/hooks/package.json

```diff
@@ -4,17 +4,18 @@
     "version": "1.0.0",
     "main": "src/index.ts",
     "dependencies": {
+        "@binary-com/binary-document-uploader": "^2.4.8",
         "@deriv/api": "^1.0.0",
         "@deriv/stores": "^1.0.0",
         "@deriv/utils": "^1.0.0",
         "@deriv/shared": "^1.0.0",
         "react": "^17.0.2",
-        "@binary-com/binary-document-uploader": "^2.4.8"
+        "usehooks-ts": "^2.7.0"
```

Comment:
Why do we need a library for using a localstorage?

Reply:
You're right. I found using useLocalStorage easy, so I went with that. Instead of installing the whole package, I think it's better to just copy the hook from their docs and use it as a custom hook.will update the code